activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -290,14 +290,18 @@ def internal_metadata # :nodoc:
       # Retrieve the connection associated with the current thread, or call
```

Comment:
Would it be worth updating the documentation, i.e. is `current thread` here correct w.r.t. `ActiveSupport::IsolatedExecutionState`?